app/models/search.rb

```diff
@@ -21,7 +23,11 @@ def initialize(params = {})
       send("#{key}=", val) if respond_to?("#{key}=")
     end
     if source == :person
-      @scope = Person.joins(:family)
+      if params[:group_category]
+        @scope = Person.eager_load(:family, :groups) # for left outer join for groups
```

Comment:
it's the eager_load here that works I think to get a LEFT OUTER JOIN but do not know what the affect would be for the join to :family